### ISSUE TITLE
After install from setup.py, execution fails with missing MyImage and ModuleNotFound error

### ISSUE THREAD

**Issue by texastwister** [rst2pdf/rst2pdf#789]
**While trying to update to the newest version, I may have corrupted something.  The current situation is as follows:  I pulled the latest updates from the Git repo and installed with "sudo python setup.py install" which appears to complete without error.

When I run rst2pdf by itself, or with --help, it errors out with the following:
Traceback (most recent call last):
  File "/usr/local/bin/rst2pdf", line 11, in <module>
    load_entry_point('rst2pdf==0.95.dev0', 'console_scripts', 'rst2pdf')()
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 489, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2843, in load_entry_point
    return ep.load()
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2434, in load
    return self.resolve()
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2440, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/local/lib/python2.7/dist-packages/rst2pdf-0.95.dev0-py2.7.egg/rst2pdf/createpdf.py", line 88, in <module>
    from rst2pdf.image import MyImage, missing
  File "/usr/local/lib/python2.7/dist-packages/rst2pdf-0.95.dev0-py2.7.egg/rst2pdf/image.py", line 24, in <module>
    except ModuleNotFoundError:
NameError: name 'ModuleNotFoundError' is not defined

I think I see the problem, after running the pip freeze command you request below -- my system still seems to be trying to call an older version of rst2pdf.  But I'm too much a novice at python to know how to fix it.

**

**What is the expected output? What do you see instead?**

🖥 **Versions**

`python -V`
Python 2.7.12

`pip freeze | grep rst2pdf`
/usr/local/lib/python2.7/dist-packages/pip/_vendor/requests/__init__.py:83: RequestsDependencyWarning: Old version of cryptography ([1, 2, 3]) may cause slowdown.
  warnings.warn(warning, RequestsDependencyWarning)
DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date. A future version of pip will drop support for Python 2.7.
rst2pdf==0.94.1

`pip freeze | grep reportlab`
/usr/local/lib/python2.7/dist-packages/pip/_vendor/requests/__init__.py:83: RequestsDependencyWarning: Old version of cryptography ([1, 2, 3]) may cause slowdown.
  warnings.warn(warning, RequestsDependencyWarning)
DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date. A future version of pip will drop support for Python 2.7.
reportlab==3.5.23

`pip freeze | grep Sphinx`

(Optional) wordaxe version :

Which operating system are you using?

    Linux Mint 18.3 Sylvia

**Additional information**

**Comment by texastwister** [comment]
I used pip to uninstal the old version of rst2pdf, then reinstalled the new version.  Now I get this: 

$ pip freeze | grep rst2pdf
/usr/local/lib/python2.7/dist-packages/pip/_vendor/requests/__init__.py:83: RequestsDependencyWarning: Old version of cryptography ([1, 2, 3]) may cause slowdown.
  warnings.warn(warning, RequestsDependencyWarning)
DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date. A future version of pip will drop support for Python 2.7.
rst2pdf==0.95.dev0

but still get this:

$ rst2pdf --help
Traceback (most recent call last):
  File "/usr/local/bin/rst2pdf", line 11, in <module>
    load_entry_point('rst2pdf==0.95.dev0', 'console_scripts', 'rst2pdf')()
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 489, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2843, in load_entry_point
    return ep.load()
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2434, in load
    return self.resolve()
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2440, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/local/lib/python2.7/dist-packages/rst2pdf-0.95.dev0-py2.7.egg/rst2pdf/createpdf.py", line 88, in <module>
    from rst2pdf.image import MyImage, missing
  File "/usr/local/lib/python2.7/dist-packages/rst2pdf-0.95.dev0-py2.7.egg/rst2pdf/image.py", line 24, in <module>
    except ModuleNotFoundError:
NameError: name 'ModuleNotFoundError' is not defined

**Comment by stephenfin** [comment]
Looks like this was introduced in d0ab3554c and is a legit bug. `ModuleNotFoundError` is a Python 3-only exception.

```
$ python2
>>> ModuleNotFoundError
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'ModuleNotFoundError' is not defined
```

```
$ python3
>>> ModuleNotFoundError
<class 'ModuleNotFoundError'>
```

We can replace that with `ImportError`, which is a base of `ModuleNotFoundError`.

```python
>>> ModuleNotFoundError.__bases__
(<class 'ImportError'>,)
```

**Comment by akrabat** [comment]
Fixed in #800
